src/components/tl/ucp/tl_ucp_team.c

```diff
@@ -86,6 +121,19 @@ UCC_CLASS_INIT_FUNC(ucc_tl_ucp_team_t, ucc_base_context_t *tl_context,
                  "topo is not available, disabling ranks reordering");
         self->cfg.use_reordering = 0;
     }
+    if (self->topo && !IS_SERVICE_TEAM(self) && self->cfg.calc_opt_radix) {
+        status = ucc_sbgp_create(self->topo, UCC_SBGP_NODE_LEADERS);
+        if (UCC_OK != status) {
+            ucc_ep_map_destroy_nested(&self->ctx_map);
+            ucc_topo_cleanup(self->topo);
+            return status;
+        }
+        max_radix       = ucc_min(UCC_TL_TEAM_SIZE(self),
+                                  self->topo->min_socket_size);
```

Comment:
don't access it directly instead pls add function ucc_topo_min_socket_size similar to ucc_topo_min_ppn